lib/traject/config/geo_aardvark_config.rb

```diff
@@ -242,8 +242,9 @@ def log_skip(context)
 # - links items that were georeferenced to their original version
 to_field 'dct_source_sm', cocina_descriptive('relatedResource'), select_type('has other format'),
          select(->(res) { res['displayLabel'] == 'Scanned map' }),
-         transform(->(res) { res.fetch('identifier', []).dig(0, 'value') }),
-         transform(->(purl) { "stanford-#{purl.split('/').last}" if purl })
+         transform(->(res) { res.fetch('identifier', []).dig(0, 'value')&.split('/')&.last }),
+         select(->(id) { id.match?(/[a-z]{2}\d{3}[a-z]{2}\d{4}/) if id }),
```

Comment:
Just for posterity could you comment something like "extracts druid from purl and constructs earthworks link" ?

Reply:
Good call. done